### DISCUSSION TITLE
MS Teams alert title template

### DISCUSSION THREAD

**Discussion by misastovicek** [grafana/grafana#44222] [Alerting]
**What would you like to be added:**
We would like to be able to define custom template for title of the MS Teams card. Currently there is hardcoded default template

**Why is this needed:**
It would be nice to have labels only in the message body instead of both the title and the body.

**Comment by mucho-al-dente** [comment]
This has been addressed in e8f4b58a8ba40812c938942442ca8fb552a9360f and merged into `main`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
